docs/guides/integration-prebid-server-side.md

```diff
@@ -0,0 +1,417 @@
+
+For details, see [Unified ID 2.0 Configuration](https://docs.prebid.org/dev-docs/modules/userid-submodules/unified2.html#unified-id-20-configuration) in the Prebid documentation.
+
+```js
+pbjs.setConfig({ 
+  userSync: { 
+    userIds: [{ 
+      name: 'uid2', 
+      params: { 
+
+                 //default value is ‘localStorage’ 
+        storage: ‘cookie’  
+      } 
+    }] 
+  } 
+}); 
+```
+
+The cookie size can be significant, which could be a problem. However, if local storage is not an option, this is one possible approach.
+
+## Determining Whether the Module Has a Valid Token
+
+You can do a check to determine whether the Prebid.js module has a valid token or you need to provide a new one.
+
+To do this, check the value returned by `pbjs.getUserIds().uid2`, as shown in the following example:
+
+```js
+const params = {};
+
+if (!pbjs.getUserIds().uid2) {
+  // There is no token that can be used or refreshed.
+  // Configure the UID2 module with a new token
+}
+
+pbjs.setConfig({
+  userSync: {
+    userIds: [{
+      name: 'uid2',
+      params: {
+        uid2Token: {
+          'advertising_token': '...advertising token...',
+          'refresh_token': '...refresh token...',
+          // etc. - see the sample token for contents of this object
+        }
+      }
+    }]
+  }
+});
+```
+
+:::caution
+If you configure a user ID by calling `setConfig` (or any similar function) twice, you will need to call `refreshUserIds` for the user ID submodules, to reinitialize their ID values.
+:::
+
+## Checking the Integration
+
+To check that the UID2 module has a valid UID2 token, call `pbjs.getUserIds().uid2`. If a value is returned, a valid UID2 token still exists in the UID2 module.
+
+If there are problems with the integration, here are some steps you can take:
+
+- Check the browser console logs.
+- Use the browser developer tools to inspect the API calls to the UID2 service.
+
+For additional help, refer to Prebid's documentation on [Troubleshooting Prebid.js](https://docs.prebid.org/troubleshooting/troubleshooting-guide.html) and [Debugging Prebid.js](https://docs.prebid.org/debugging/debugging.html).
+
+An example of a tool for validating and debugging Prebid.js configuration is Professor Prebid, an open-source Chrome extension:
+
+- Chrome web store download location: [Professor Prebid](https://chromewebstore.google.com/detail/professor-prebid/kdnllijdimhbledmfdbljampcdphcbdc)
+- Documentation on prebid.org: [Professor Prebid User Guide](https://docs.prebid.org/tools/professor-prebid.html)
+
+## Configuration Parameters for userSync
+
+The following parameters apply only to the UID2 Prebid User ID Module integration.
+
+(**GWH_SW question I missed asking you. When we say "Optional, client refresh" -- what does that mean? Is it required for client refresh? Or N/A for server-only and optional for client refresh? And will readers know? If something is required for a specific mode we should say that.**)
```

Comment:
"Optional, client refresh mode" means It's optional in Client Refresh Mode only and not required (or even optional) for server only mode at all

Reply:
Thx much.